src/Adapter/File/Uploader/AttachmentFileUploader.php

```diff
@@ -133,23 +128,14 @@ private function uploadFile(string $filePath, string $uniqid, int $fileSize): vo
     }
 
     /**
-     * @param int $fileSize
-     *
      * @throws AttachmentConstraintException
      */
     private function checkFileAllowedForUpload(int $fileSize): void
     {
         $maxFileSize = $this->uploadSizeConfiguration->getMaxUploadSizeInBytes();
 
         if ($maxFileSize > 0 && $fileSize > $maxFileSize) {
-            throw new AttachmentConstraintException(
-                sprintf(
-                    'Max file size allowed is "%s" bytes. Uploaded file size is "%s".',
-                    $maxFileSize,
-                    $fileSize
-                ),
-                AttachmentConstraintException::INVALID_FILE_SIZE
-            );
+            throw new AttachmentConstraintException(sprintf('Max file size allowed is "%s" bytes. Uploaded file size is "%s".', $maxFileSize, $fileSize), AttachmentConstraintException::INVALID_FILE_SIZE);
```

Comment:
It was so much easier to read when it was not on one line, is there a reason?

Reply:
no, can it be cs fixer? Ill check

Reply:
Yup i checked, cs fixer puts it in one line :octocat:

Reply:
oh ok...  👍

Reply:
Its strange anyway, because I've pushed the multiline code without fixing and prettyCI passes.

Reply:
Yeah it's a pain this new modif, I wonder if it's not related to `printf` rather than the exception

Reply:
I don't agree with this change. If cs fixer is doing it then it's misconfigured because it violates PSR-2 line length for no reason.